config/v201/config.json

```diff
@@ -776,6 +775,12 @@
                     "Actual": ""
                 }
             },
+            "SmartChargingCtrlrAvailable": {
```

Comment:
The setting that we need has already been defined in the config. The spec indicates that this field should be called `Enabled`:
- see Part 2, "Referenced Components and Variables", §2.9.1, pages 458–459 for the definition
- see, for instance, the Notes column of K05.FR.04–05 for direct commentary on this being the correct setting to employ

Based on the `config.json`, the `Enabled` variable is defined in the next entry in this map, with key `SmartChargingCtrlrAvailableEnabled`.

Reply:
Now, _why_ that key appears to be an amalgamation of the `SmartChargingEnabled` and `SmartChargingAvailable` variables from the device model is unclear, but it doesn't have any bearing on its use for this card. Perhaps @Pietfried can shed some light here?

Reply:
Wait … despite `SmartChargingEnabled` being called out in K05.FR.04–05 as how to determine if smart charging is supported, the description for `SmartChargingAvailable` in §2.9.2 is "Whether Smart Charging is supported."

… _sigh_.

Reply:
I think it makes sense to still key off of `SmartChargingCtrlrAvailableEnabled` rather than `SmartChargingCtrlrAvailable`, since it seems to be the more relevant value. I believe that …

- `Available` indicates whether or not a given Charging Station's HW/SW can support smart charging.
- `Enabled` indicates whether or not smart charging capabilities are switched on at any given time.

This is mostly based on the fact that `Enabled` is mutable while `Available` is immutable at runtime.

Reply:
We should be keying off of the `Enabled` variable, even if it doesn't provide the best user experience. For a detailed discussion of the topic, check out [this post on Zulip](https://lfenergy.zulipchat.com/#narrow/stream/417676-EVerest.3A-Cloud-communication/topic/Proper.20Use.20of.20Accepted.20and.20Enabled.20Controller.20Variables/near/460109230).